services/explorer-indexer/src/modules/health/health.controller.ts

```diff
@@ -6,7 +6,7 @@ import {
   DiskHealthIndicator,
   MemoryHealthIndicator,
 } from '@nestjs/terminus';
-import { SyncerHealthIndicator } from './SyncerHealthIndicator';
+import { SyncerHealthIndicator } from './syncerHealthIndicator';
```

Comment:
Why is this camel case ? The file name is in pascal case.

Reply:
To make it uniform with the other namings